psycopg/psycopg/_typeinfo.py

```diff
@@ -22,6 +22,7 @@
 
 T = TypeVar("T", bound="TypeInfo")
 RegistryKey: TypeAlias = Union[str, int, Tuple[type, int]]
+_AnyConn = Union["Connection[Any]", "AsyncConnection[Any]"]
```

Comment:
```suggestion
_AnyConn: TypeAlias = Union["Connection[Any]", "AsyncConnection[Any]"]
```

with its import

Reply:
Is this needed by some type checkers?

I've updated but I don't think mypy complains about it.